### DISCUSSION TITLE
Report false positives for Phishing

### DISCUSSION THREAD

**Discussion by tiagozip** [codesandbox/codesandbox-client#6764] [General] [answered]
Just to simplify things I have created this thread to report false positives for Phishing.
Please note that this is **not** official so this is just somewhere you can put your reports and wait for @codesandbox to read them

**Comment by mirko314** [comment]
Thank you, just reposting the same URL posted in the other issue:
We use CSB for our open source examples and at least the following one got flagged as well as Phishing unfortunately.
See e.g. https://codesandbox.io/s/github/imgly/cesdk-web-examples/tree/v1.6.3/configure-presets-templates .
Github: https://github.com/imgly/cesdk-web-examples/tree/v1.6.3/configure-presets-templates
The examples are rather simple, so I do not know why exactly this one got flagged.

**Comment by deepakpalrocks** [comment]
https://codesandbox.io/s/keen-sky-nni27j
My sandbox that I have created for my college project. Could you please remove it from 'potential phising pages'. I even changed the name when I found out there was an android app with similar name, its UI is not the same as of another existing app as of my knowledge.

**Comment by tiagozip** [comment, marked as answer]
Closed. For false phishing reports please email support@codesandbox.io (official announcement)
